.github/workflows/nightly.yaml

```diff
@@ -25,6 +25,16 @@ jobs:
           packager_tag: el8-go1.17
           version: nightly
 
+  bullseye_aarch64:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build
+        uses: choria-io/actions/packager@main
+        with:
+          build_package: bullseye_aarch64
+          packager_tag: el8-go1.17
```

Comment:
bullseye-go1.17 Here

Reply:
Also add it to the dependency list a bit lower in tbe file

Reply:
Every other job seems to use the `el*-go1.17` tag though? Should the bullseye build be unique in that regard?

Reply:
every other job builds rpms - this one will build a deb to need the deb stuff.

Reply:
Even the Windows zip and the MacOS jobs use the `el8-go1.17` tag though, that's my confusion

Reply:
the go build can happen on anything - go cross compiles from anywhere to anywhere. Those other jobs just need zip/tar nothing OS specific.

But to build a deb we need debian machine so that swaps it over to debian and then all the things to make debs are there.

I will kick a nightly now and we can see what happens, tbf, this stuff is confusing, i hate CI :)